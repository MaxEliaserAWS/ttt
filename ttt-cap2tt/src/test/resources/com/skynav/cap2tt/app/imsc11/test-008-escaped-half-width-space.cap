Lambda字幕V4	DF0+1	SCENE"和文標準"

40	00015700/00020000	_h_a_l_f_
				half
