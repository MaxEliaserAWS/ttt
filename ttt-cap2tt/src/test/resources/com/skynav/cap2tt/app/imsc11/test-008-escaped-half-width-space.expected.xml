<?xml version="1.0" encoding="utf-8"?><tt xmlns="http://www.w3.org/ns/ttml" ttp:contentProfiles="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttp:frameRate="30" ttp:profile="http://www.w3.org/ns/ttml/profile/imsc1.1/text" ttva:model="imsc11" xml:lang="ja" xmlns:ebutts="urn:ebu:tt:style" xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling" xmlns:ttva="http://skynav.com/ns/ttv/annotations"><head><styling><initial tts:fontSize="4rh"/><initial tts:lineHeight="5rh"/><initial tts:showBackground="whenActive"/><style xml:id="s1" ebutts:multiRowAlign="start" tts:textAlign="center"/></styling><layout><region xml:id="横下" tts:displayAlign="after" tts:extent="80rw 15rh" tts:position="center bottom 10rh"/></layout></head><body region="横下" xml:space="preserve"><div><p begin="00:01:57:00" end="00:02:00:00" style="s1"> h a l f <br/>half</p></div></body></tt>